exporter/metadata.py

```diff
@@ -69,14 +108,23 @@ def provenance_from_dict(data: dict):
 
             # Populate the major and minor schema versions from the URL in the describedBy field
             schema_semver = re.findall(r'\d+\.\d+\.\d+', data["content"]["describedBy"])[0]
-            schema_major_version = int(schema_semver.split(".")[0])
-            schema_minor_version = int(schema_semver.split(".")[1])
-
-            return MetadataProvenance(uuid, submission_date, update_date, schema_major_version,
-                                      schema_minor_version)
+            concrete_type = data['content']['describedBy'].rsplit('/', 1)[-1]
+            version_with_schema_fields = SCHEMA_VERSIONS_WITHOUT_SCHEMA_FIELDS.get(concrete_type)
+            if MetadataResource.version_has_schema_fields(schema_semver, version_with_schema_fields):
+                schema_major_version = int(schema_semver.split(".")[0])
```

Comment:
tiny: `schema_major_version, schema_minor_version = [int(x) for x in schema_semver.split(".")]`

Take it or leave it, I like one liners

Reply:
Nice one

Reply:
Applied with a slight modification schema_major_version, schema_minor_version, _ = [int(x) for x in schema_semver.split(".")]  since schema_semver is a semantic version and has 3 parts

Reply:
`schema_major_version, schema_minor_version = [int(x) for x in schema_semver.split(".")][:2]` would be better then but also take it or leave it. It's not a big thing